### ISSUE TITLE
False positives

### ISSUE THREAD

**Issue by greew** [wix/eslint-plugin-lodash#124]
Hi maintainers.

I've been troubling with getting false positives in when trying to lint my JS. 

I ended up creating a simple file `test.js` with only the following code:

```javascript
var func = function (a) {
    return a;
};
_.map([], func);
```
which results in 

```bash
08:48 $ eslint test.js

/home/jesper/code/test/test.js
  4:1  error  Prefer '_.map' over the native function  lodash/prefer-lodash-method

✖ 1 problem (1 error, 0 warnings)
```

My setup is as follows: 

```
08:48 $ npm -v
3.5.2

08:48 $ nodejs -v
v4.2.6

08:48 $ nodejs list -g
...
├─┬ eslint@3.10.2
...
├─┬ eslint-plugin-lodash@2.2.0
│ └── lodash@4.17.2
...

08:58 $ cat package.json 
{
  ...
  "dependencies": {
    "gulp": "^3.9.1",
    "jquery": "^3.1.0",
    "laravel-elixir": "^6.0.0-9",
    "laravel-elixir-lodash": "^3.0.6",
    "lodash": "^4.14.0"
  },
  "devDependencies": {
    "laravel-elixir-browsersync-official": "^1.0.0"
  },
  "eslintConfig": {
    "plugins": ["lodash"],
    "globals": {
      "Bloodhound": true,
      "PNotify": false,
      "WSD": true,
      "_": false,
      "Page": true,
      "bootbox": false
    },
    "extends": [
      "eslint:recommended",
      "plugin:lodash/recommended"
    ],
    "rules": {
      "linebreak-style": ["error", "unix"],
      "semi": ["error", "always"],
      "comma-dangle": ["error", "always-multiline"],
      "no-cond-assign": ["error", "always"],
      "no-console": "off",
      "lodash/prefer-lodash-method": [2, {"ignoreMethods": ["find", "filter", "includes"]}]
    },
    "env": {
      "jquery": true,
      "browser": true
    }
  }
}
```

Can you help me find out what is wrong here, because I'm running headfirst into a wall whatever I'm trying here :smile: 

If you need any extra info or cannot help me, just drop a note, and I'll close the issue.

Best regards
/Jesper

**Comment by ganimomer** [comment]
Hey,
Since version 2.0, the plugin's `recommended` config assumes you use CommonJS or ES6 modules for importing Lodash or its individual functions.
If you want to use `_` as a global, you should [use the `canonical` config](https://github.com/wix/eslint-plugin-lodash#configuration-for-use-with-the-full-lodash-object), or [define a `pragma` setting yourself](https://github.com/wix/eslint-plugin-lodash#shared-rule-settings).

**Comment by greew** [comment]
@ganimomer And using the `canonical` config did the trick!

Thanks a lot for a quick answer!! :-)
